shared/AppInsightsCommon/src/UrlHelperFuncs.ts

```diff
@@ -65,7 +65,7 @@ export function urlGetCompleteUrl(method: string, absoluteUrl: string) {
 
 // Fallback method to grab host from url if document.createElement method is not available
 export function urlParseHost(url: string, inclPort?: boolean) {
-    let fullHost = urlParseFullHost(url, inclPort);
+    let fullHost = urlParseFullHost(url, inclPort) || "";
```

Comment:
Add `| ""` means when url has no host, the `urlParseFullHost` method returns `null`. In that case, `null` passed back to [here](https://github.com/microsoft/ApplicationInsights-JS/blob/2612a3cc285526fb66464f860d049ba3d145180b/shared/AppInsightsCommon/src/Util.ts#L351) but `null.toLowerCase()` is wrong. So always return an empty string when host is not detected.